examples/host_filter_dag.json

```diff
@@ -6,8 +6,8 @@
     "priceseq_out": ["priceseq1.fq", "priceseq2.fq"],
     "cdhitdup_out": ["dedup1.fa", "dedup2.fa"],
     "lzw_out": ["lzw1.fa", "lzw2.fa"],
-    "bowtie_out": ["bowtie1.fa", "bowtie2.fa"],
-    "gsnap_filter_out": ["gsnap_filtere1.fa", "gsnap_filtere2.fa"]
+    "bowtie_out": ["bowtie2_1.fa", "bowtie2_2.fa", "bowtie2_merged.fa"],
```

Comment:
I think it would be nice if we made it a convention to write “bowtie” and “rapsearch” instead of “bowtie2” and “rapsearch2” (and consistently “gsnap” instead of “gsnapl”)

Reply:
Brian Yu told me Bowtie2 is a very different beast from Bowtie. Perhaps we can change all the wording to bowtie2 so when people look at our code, they can be sure we are running bowtie2.